### PR TITLE
Added missed commit from Babysitter to child-view-container

### LINKED ISSUE #3099: babysitter PR not in pre5
A commit to the major branch of babysitter was overlooked moving it to Mn.

https://github.com/marionettejs/backbone.babysitter/pull/73/files

@rafde interested in adding this to `next`?

### PR DESCRIPTION
**Pull request by ahumphreys87**
Fixes #3099

### CODE REVIEW

File: src/child-view-container.js [hunk 1]
```diff
@@ -84,24 +97,42 @@ _.extend(Container.prototype, {
...
 
-    // update the length
-    this._updateLength();
     return this;
   },
 
+  // Call a method on every view in the container,
+  // passing parameters to the call method one at a
+  // time, like `function.call`.
+  call(method) {
```
Comment (paulfalgout) [inline src/child-view-container.js]: call and apply can be removed.  They don't actually do anything you can't do with the built in underscore functions:  https://github.com/marionettejs/backbone.babysitter/issues/57

Comment (paulfalgout) [inline src/child-view-container.js]: subsequently this is also the coverage issue.

Comment (rafde) [inline src/child-view-container.js]: I don't know why test/unit/child-view-container.spec.js is included in marionette when the test are for babysitter. Babysitter shouldn't be part of backbone.marionette's coverage.

Comment (paulfalgout) [inline src/child-view-container.js]: recent change to remove the bundled build made babysitter part of Mn.  So babysitter is no longer an external dependency

Comment (rafde) [inline src/child-view-container.js]: ahh... ok.

**Comment by ahumphreys87** [comment]
@paulfalgout updated

**Comment by paulfalgout** [comment]
:+1:

**Comment by justinffs** [comment]
This merge does not seem to have a build.  See https://github.com/marionettejs/backbone.marionette/blob/next/lib/backbone.marionette.js#L1970

[_.any() has been removed from lodash 4](https://github.com/lodash/lodash/wiki/Changelog#notable-changes)

**Comment by paulfalgout** [comment]
@justinffs that's correct.. it'll make 3.0 which'll be out super shortly.

**Pull request merged**